src/main/plugins/org.dita.pdf2/cfg/common/vars/en.xml

```diff
@@ -29,6 +29,9 @@ with those set forth herein.
 This file is part of the DITA Open Toolkit project. 
 See the accompanying LICENSE file for applicable license.
 -->
+
+<!-- 20170503 SCH: Add support for troubleshooting elements. -->
+
```

Comment:
Yeah, those still linger. FWIW those are leftover from ... wow, almost 8 years ago ... years before we were in github, and when the overall source control was mostly handled by a rotating set of folks in IBM China. We just haven't cleaned those out.